saw-core/src/Verifier/SAW/Term/Pretty.hs

```diff
@@ -442,8 +462,8 @@ ppTermF prec (Pi x tp body) =
   ppParensPrec prec PrecLambda <$>
   (ppPi <$> ppTerm' PrecApp tp <*>
    ppTermInBinder PrecLambda x body)
-ppTermF _ (LocalVar x) = (text <$> varLookupM x) >>= maybeColorM dullgreen
-ppTermF _ (Constant ec _) = maybeColorM dullblue $ ppName (ecName ec)
+ppTermF _ (LocalVar x) = (text <$> varLookupM x) >>= maybeColorM (colorDull Green)
+ppTermF _ (Constant ec _) = maybeColorM (colorDull Blue) $ ppName (ecName ec)
```

Comment:
This would be a great place to `reAnnotate` from the descriptive annotations to the presentation annotations, rewriting `PrConstant` to `colorDull Blue` and such.  The additional benefit is that different outputs (e.g. non-ANSI terminals) could us a different mapping, and or ignore the annotations entirely and emit just plain text.

Reply:
Yeah, that's a good idea. It would make the intention of the code more clear. Being able to easily remap the styles would be nice too. And it would also lower the barrier for extending the annotation type to do more semantic markup, since I wouldn't have to choose the color once-and-for-all at the beginning :)